Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -54,7 +54,51 @@ public function testAutowiringAlias()
 
             $alias = $container->getAlias($id);
             $this->assertEquals($target, (string) $alias, sprintf('The autowiring for `%s` should use `%s`.', $id, $target));
-            $this->assertFalse($alias->isPublic(), sprintf('The autowiring alias for `%s` should be private.', $id, $target));
+            $this->assertFalse($alias->isPublic(), sprintf('The autowiring alias for `%s` should be private.', $id));
+        }
+    }
+
+    public function testEntityManagerAutowiringAlias()
+    {
+        if (! interface_exists(EntityManagerInterface::class)) {
+            self::markTestSkipped('This test requires ORM');
+        }
+
+        if (! method_exists(ContainerBuilder::class, 'registerAliasForArgument')) {
+            self::markTestSkipped('Test requires Symfony 4.2 or higher');
+        }
+
+        $container = $this->getContainer([
+            'YamlBundle',
+            'XmlBundle',
+        ]);
+        $extension = new DoctrineExtension();
+
+        $config = BundleConfigurationBuilder::createBuilder()
+            ->addBaseConnection()
+            ->addEntityManager([
+                'entity_managers' => $entityManagers = [
+                    'default' => [
+                        'mappings' => ['YamlBundle' => []],
+                    ],
+                    'logs' => [
```

Comment:
Could you please use a snake_cased_name here? I'm curious how this plays out, as I'd expect a name like `purchase_logs` being mapped to `$purchaseLogsEntityManager`, so I think it's worth confirming this.

Reply:
@alcaeus Done, works as you expected it would :smile: